lib/SimpleSAML/Resources/config/routes/routes.yaml

```diff
@@ -1,6 +0,0 @@
-remove_trailing_slash:
-    path: /{url}
-    defaults: { _controller: SimpleSAML\Module:removeTrailingSlash }
-    requirements:
-        url: .*/$
-    methods: [GET]
```

Comment:
This broke the access to `module.php/admin/` and I haven't discovered an added value yet